<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAppClass</class>
 <widget class="QMainWindow" name="MyAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleMesh</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLCanvas" name="glcanvas" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFit"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionPanLeft"/>
   <addaction name="actionPanRight"/>
   <addaction name="actionPanDown"/>
   <addaction name="actionPanUp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="modelingToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionGrid"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="separator"/>
   <addaction name="actionPolyLine"/>
   <addaction name="actionQuadBezier"/>
   <addaction name="actionCubicBezier"/>
   <addaction name="actionCircle"/>
   <addaction name="actionCircleArc"/>
   <addaction name="actionDelete"/>
   <addaction name="actionIntersect"/>
   <addaction name="actionCreateRegion"/>
  </widget>
  <widget class="QToolBar" name="meshToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionBoundaryMesh"/>
   <addaction name="actionDomainMesh"/>
  </widget>
  <action name="actionFit">
   <property name="icon">
    <iconset>
     <normaloff>icons/fit.jpg</normaloff>icons/fit.jpg</iconset>
   </property>
   <property name="text">
    <string>Fit</string>
   </property>
   <property name="toolTip">
    <string>Adjust object image to fit on canvas</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoomIn2.jpg</normaloff>icons/zoomIn2.jpg</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>Zoom in object image on canvas</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoomout2.jpg</normaloff>icons/zoomout2.jpg</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom out object image on canvas</string>
   </property>
  </action>
  <action name="actionPanLeft">
   <property name="icon">
    <iconset>
     <normaloff>icons/arrowLeft2.jpg</normaloff>icons/arrowLeft2.jpg</iconset>
   </property>
   <property name="text">
    <string>PanLeft</string>
   </property>
   <property name="toolTip">
    <string>Translate left object image on canvas</string>
   </property>
  </action>
  <action name="actionPanRight">
   <property name="icon">
    <iconset>
     <normaloff>icons/arrowrRight.jpg</normaloff>icons/arrowrRight.jpg</iconset>
   </property>
   <property name="text">
    <string>PanRight</string>
   </property>
   <property name="toolTip">
    <string>Translate right object image on canvas</string>
   </property>
  </action>
  <action name="actionPanDown">
   <property name="icon">
    <iconset>
     <normaloff>icons/arrowrDown.jpg</normaloff>icons/arrowrDown.jpg</iconset>
   </property>
   <property name="text">
    <string>PanDown</string>
   </property>
   <property name="toolTip">
    <string>Translate down object image on canvas</string>
   </property>
  </action>
  <action name="actionPanUp">
   <property name="icon">
    <iconset>
     <normaloff>icons/arrowup2.jpg</normaloff>icons/arrowup2.jpg</iconset>
   </property>
   <property name="text">
    <string>PanUp</string>
   </property>
   <property name="toolTip">
    <string>Translate up object image on canvas</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/mouse3.png</normaloff>icons/mouse3.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Set selection mode</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/line.png</normaloff>icons/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Insert a straight line segment</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>icons/delete3.png</normaloff>icons/delete3.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected entities</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/grid2.png</normaloff>icons/grid2.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Specity grid points</string>
   </property>
  </action>
  <action name="actionIntersect">
   <property name="icon">
    <iconset>
     <normaloff>icons/intersect2.png</normaloff>icons/intersect2.png</iconset>
   </property>
   <property name="text">
    <string>Intersect</string>
   </property>
   <property name="toolTip">
    <string>Intersect two selected curves</string>
   </property>
  </action>
  <action name="actionCreateRegion">
   <property name="icon">
    <iconset>
     <normaloff>icons/region.png</normaloff>icons/region.png</iconset>
   </property>
   <property name="text">
    <string>CreateRegion</string>
   </property>
   <property name="toolTip">
    <string>Create region from selected curves</string>
   </property>
  </action>
  <action name="actionBoundaryMesh">
   <property name="icon">
    <iconset>
     <normaloff>icons/boundarymesh2.png</normaloff>icons/boundarymesh2.png</iconset>
   </property>
   <property name="text">
    <string>BoundaryMesh</string>
   </property>
   <property name="toolTip">
    <string>Specify curve number of subdivisions</string>
   </property>
  </action>
  <action name="actionDomainMesh">
   <property name="icon">
    <iconset>
     <normaloff>icons/mesh.png</normaloff>icons/mesh.png</iconset>
   </property>
   <property name="text">
    <string>DomainMesh</string>
   </property>
   <property name="toolTip">
    <string>Generate mesh in region</string>
   </property>
  </action>
  <action name="actionPolyLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/poly2.png</normaloff>icons/poly2.png</iconset>
   </property>
   <property name="text">
    <string>PolyLine</string>
   </property>
   <property name="toolTip">
    <string>Insert a polyline</string>
   </property>
  </action>
  <action name="actionQuadBezier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/quadbezier.png</normaloff>icons/quadbezier.png</iconset>
   </property>
   <property name="text">
    <string>QuadBezier</string>
   </property>
   <property name="toolTip">
    <string>Insert a quadbezier</string>
   </property>
  </action>
  <action name="actionCubicBezier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/cubicbezier2.png</normaloff>icons/cubicbezier2.png</iconset>
   </property>
   <property name="text">
    <string>CubicBezier</string>
   </property>
   <property name="toolTip">
    <string>Insert a cubicbezier</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/circle.png</normaloff>icons/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>Insert a circle</string>
   </property>
  </action>
  <action name="actionCircleArc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/circlearc.png</normaloff>icons/circlearc.png</iconset>
   </property>
   <property name="text">
    <string>CircleArc</string>
   </property>
   <property name="toolTip">
    <string>Insert a circle arc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLCanvas</class>
   <extends>QWidget</extends>
   <header>glcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myapp.qrc"/>
 </resources>
 <connections/>
</ui>
